### ISSUE TITLE
Website is gone again!

### ISSUE THREAD

**Issue by vsivsi** [skosch/Crimson#16]
http://aldusleaf.org/crimson.html doesn't work.  First a certificate error, then a 404.

Related to: https://github.com/skosch/Crimson/issues/14

**Comment by skosch** [comment]
Yeah ... I've been messing around with my mailserver, and lost the nginx configuration in the process. I think it's time we move the website's content to the readme

**Comment by vsivsi** [comment]
Sounds that way...  Or maybe create a github wiki for the project: https://github.com/skosch/Crimson/wiki

**Comment by Jonta** [comment]
Updates on this?
